docs/shared/spm-installation-panel.mdx

```diff
@@ -6,42 +6,46 @@ import {
 
 <ExpansionPanel title="Swift Package Manager Installation">
 
-**NOTE**: These instructions are intended for usage on Xcode 11 and higher. Xcode 11 is the first version of Xcode that integrates Swift Package manager and makes it _way_ easier to use than it was at the command line. If you are using older versions of Xcode, we recommend using CocoaPods.
+> **Note:** These instructions are for Xcode 11 or later. Xcode 11 is the first version of Xcode that integrates Swift Package manager, whereas older versions require using the command line. If you're using an older version of Xcode, we recommend using CocoaPods instead.
 
 <ExpansionPanelList>
 <ExpansionPanelListItem number="1">
 
 Go to **File > Swift Packages > Add Package Dependency...**
 
-![add a dependency](../source/screenshot/spm_packages_add_dependency.png)
+<img class="screenshot" src="../source/screenshot/spm_packages_add_dependency.png" alt="Adding an SPM dependency" />
 
 </ExpansionPanelListItem>
 <ExpansionPanelListItem number="2">
 
-Paste the URL to the Apollo iOS repo on GitHub ([https://github.com/apollographql/apollo-ios.git](https://github.com/apollographql/apollo-ios.git)) into the search bar, then hit the **Next** button: 
+Paste the URL to the Apollo iOS GitHub repo (`https://github.com/apollographql/apollo-ios.git`) into the search bar, then click **Next**:
 
-![paste in the url](../source/screenshot/spm_paste_url.png)
+<img class="screenshot" src="../source/screenshot/spm_paste_url.png" alt="Pasting the Apollo iOS GitHub URL" />
```

Comment:
I can't remember if this has been updated for Xcode 13 yet but I'll take a look, the look of where you have to paste it is totally different than 12

Reply:
I've just updated Xcode myself and plan to run through the tutorial properly, I can probably update this as part of that process